DQM/Physics/python/DQMPhysics_cff.py

```diff
@@ -33,6 +34,7 @@
                            *HiggsDQM
                            *ExoticaDQM
                            *B2GDQM
+                           *CentralitypADQM
```

Comment:
OK
Still, it would be more practical to monitor only what is produced upstream.
DQM modules which silently ignore lack of input  are error-prone